.vscode/launch.json

```diff
@@ -83,6 +83,25 @@
 			"type": "node",
 			"args": ["generate", "bundleStats", "-v"],
 		},
+		{
+			"name": "flub generate releaseNotes",
+			"program": "${workspaceFolder}/build-tools/packages/build-cli/bin/dev.js",
+			"cwd": "${workspaceFolder}",
+			"request": "launch",
+			"skipFiles": ["<node_internals>/**"],
+			"type": "node",
+			"args": [
+				"generate",
+				"releaseNotes",
+				"-g",
+				"client",
+				"-t",
+				"minor",
+				"--out",
+				"RELEASE_NOTES/2.2.0.md",
```

Comment:
Do we want this hard-coded here?

Reply:
`<shrug>` I don't know. I feel like the debug comfigs are never "click and go" so some config is always expected. We can comment it out.